hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -230,6 +230,14 @@ public Response put(
       }
       LOG.error("Exception occurred in PutObject", ex);
       throw ex;
+    } catch (OS3Exception ex) {
+      if (copyHeader != null) {
+        getMetrics().incCopyObjectFailure();
+      } else {
+        LOG.error("Exception occurred in PutObject", ex.getMessage());
```

Comment:
can you explain why the error message is only logged in one case but not in another (when copyHeader is not null)?

Reply:
Thanks for pointing that out, we should be logging the error outside the if-else block so as to notify the user that the create key method has failed because an S3 exception was thrown !!